### DISCUSSION TITLE
_dialogService.Show() causes "XAML parsing failed"

### DISCUSSION THREAD

**Discussion by sigmarsson** [PrismLibrary/Prism#2966] [Uno Platform]
![image](https://github.com/PrismLibrary/Prism/assets/43340959/51a852d8-7c0c-4ef7-a2e4-46d53c37fb70)

![image](https://github.com/PrismLibrary/Prism/assets/43340959/1705a272-c0f6-4963-ad25-48e77460af37)

![image](https://github.com/PrismLibrary/Prism/assets/43340959/26f2873e-db65-486a-9d3d-8c8510d4955a)

Why ?

**Comment by sigmarsson** [comment]
btw, it works but how could I pass params in this manner ? 

![image](https://github.com/PrismLibrary/Prism/assets/43340959/bf986065-58b0-447b-9847-e7c9febfc1d4)

**Reply by dansiegel** [reply]
unfortunately without a reproduction app I can't really tell you what's going on
